usaspending_api/conftest_helpers.py

```diff
@@ -44,45 +45,55 @@ def update_index(self):
         for the index, and add contents.
         """
         self.delete_index()
-        self._refresh_materialized_views()
+        self._refresh_materialized_views(self.type)
         self.client.indices.create(self.index_name, self.template)
-        create_aliases(self.client, self.index_name, True)
+        create_aliases(self.client, self.index_name, self.type, True)
         self._add_contents()
 
     def _add_contents(self):
         """
         Get all of the transactions presented in the view and stuff them into the Elasticsearch index.
         The view is only needed to load the transactions into Elasticsearch so it is dropped after each use.
         """
-        transaction_delta_view_sql = open(
-            str(settings.APP_DIR / "database_scripts" / "etl" / "transaction_delta_view.sql"), "r"
-        ).read()
+        view_sql_file = "award_delta_view.sql" if self.type == "awards" else "transaction_delta_view.sql"
+        view_sql = open(str(settings.APP_DIR / "database_scripts" / "etl" / view_sql_file), "r").read()
         with connection.cursor() as cursor:
-            cursor.execute(transaction_delta_view_sql)
-            cursor.execute(f"SELECT * FROM {settings.ES_TRANSACTIONS_ETL_VIEW_NAME};")
+            cursor.execute(view_sql)
+            if self.type == "transactions":
+                view_name = settings.ES_TRANSACTIONS_ETL_VIEW_NAME
+            else:
+                view_name = settings.ES_AWARDS_ETL_VIEW_NAME
+            cursor.execute(f"SELECT * FROM {view_name};")
             transactions = ordered_dictionary_fetcher(cursor)
-            cursor.execute(f"DROP VIEW {settings.ES_TRANSACTIONS_ETL_VIEW_NAME};")
-
-        for transaction in transactions:
-            self.client.index(
-                self.index_name,
-                self.doc_type,
-                json.dumps(transaction, cls=DjangoJSONEncoder),
-                transaction["transaction_id"],
-            )
-
+            cursor.execute(f"DROP VIEW {view_name};")
+
+            for transaction in transactions:
```

Comment:
Doesn't cause any harm, but since this for loop doesn't use the `cursor` it doesn't need to be under the `with`.